src/routes/LPOptimisers/EcosystemDeposit/DepositForm/MaturityDistribution/MaturityDistributionToggle/MaturityDistributionToggle.tsx

```diff
@@ -0,0 +1,61 @@
+import ToggleButtonGroup from '@mui/material/ToggleButtonGroup';
+import React from 'react';
+
+import { IconLabel } from '../../../../../../components/composite/IconLabel/IconLabel';
+import { doNothing } from '../../../../../../utilities/doNothing';
+import {
+  MaturityDistributionBox,
+  MaturityDistributionInputLabel,
+  MaturityDistributionTypography,
+  ToggleButton,
+} from './MaturityDistributionToggle.styled';
+
+type Distribution = 'automatic' | 'manual';
+
+const distributions: {
+  label: string;
+  id: Distribution;
+}[] = [
+  {
+    id: 'automatic',
+    label: 'AUTOMATIC',
+  },
+  {
+    id: 'manual',
+    label: 'MANUAL',
+  },
+];
+
+export type MaturityDistributionToggleProps = {
+  distribution: Distribution;
+  onChange: (distribution: Distribution) => void;
+  disabled: boolean;
+};
+export const MaturityDistributionToggle: React.FunctionComponent<MaturityDistributionToggleProps> =
```

Comment:
I also noticed that I can click the Automatic toggle button which toggles the logic to manual but the Manual toggle is not highlighted

Reply:
@Sol3iman can you share steps to reproduce? I cannot understand it well based on comment here 🙏

Reply:
Hey Sol, feel free to share the screen recording with me and I'll incorporate the fix on develop.